x/nexus/client/cli/query.go

```diff
@@ -222,21 +227,38 @@ func getCmdChains() *cobra.Command {
 		Use:   "chains",
 		Short: "Returns the registered chain names",
 		Args:  cobra.ExactArgs(0),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientQueryContext(cmd)
-			if err != nil {
-				return err
-			}
-
-			queryClient := types.NewQueryServiceClient(clientCtx)
-
-			res, err := queryClient.Chains(cmd.Context(), &types.ChainsRequest{})
-			if err != nil {
-				return err
-			}
+	}
 
-			return clientCtx.PrintProto(res)
-		},
+	status := cmd.Flags().String("status", "", fmt.Sprintf("the chain status [%s|%s]", activated, deactivated))
```

Comment:
what about `all`?

Reply:
I removed it in #1774 since it seemed misplaced as a status. Keeping the absence of status to mean unspecified which will list all chains.